docs/hugo/content/guide/authentication/credential-format.md

```diff
@@ -40,7 +40,8 @@ helm upgrade --install --devel aso2 aso2/azure-service-operator \
         --set azureSubscriptionID=$AZURE_SUBSCRIPTION_ID \
         --set azureTenantID=$AZURE_TENANT_ID \
         --set azureClientID=$AZURE_CLIENT_ID \
-        --set azureClientSecret=$AZURE_CLIENT_SECRET
+        --set azureClientSecret=$AZURE_CLIENT_SECRET \
+        --set crdPattern='resources.azure.com/*;containerservice.azure.com/*;keyvault.azure.com/*;managedidentity.azure.com/*;eventhub.azure.com/*'
```

Comment:
Maybe mention the default (empty string) option here and leave a link to the doc where we explain how to configure  `crdPattern` value?

Reply:
I've not mentioned * explicitly here, but I've linked them to the main crdPattern page everywhere that it's used, so they can go read that to see to use `*`. That page also has the warning about `*` on Free tier clusters which felt quite verbose to repeat here N times.